package.json

```diff
@@ -10,17 +10,28 @@
   "license": "Apache 2.0",
   "unity": "2019.4",
   "unityRelease": "0f1",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "dependencies": {
     "com.unity.nuget.newtonsoft-json": "3.0.2",
-    "voxell.util": "1.4.2",
```

Comment:
I dun think we should remove the dependencies, it will mean we are having the exact same code on 2 related repos. it would be extremely hard to maintain when we start to update the util repo if you think about it, I would be writing duplicated code.